features/fields_extensie.feature

```diff
@@ -11,57 +11,116 @@ Functionaliteit: Aanpasbare representatie met de fields parameter
   - opschortingBijhouding reden en datum worden altijd meegegeven wanneer deze een waarde hebben.
```

Comment:
hier moet wel iets staan over autorisatie, want veel gebruikers zijn wel geautoriseerd voor opschortingBijhouding.reden, maar niet voor opschortingBijhouding.datum. bijv. gemeenten zijn hier voor buitengemeentelijke personen niet voor de datum geautoriseerd

Reply:
of het niet opschrijven vanuit autorisatie, maar alleen opschortingBijhouding.reden altijd leveren

Reply:
@MelvLee moet voor objecten met polymorfisme ook altijd type geleverd worden? Bijvoorbeeld bij nationaliteit, verblijfplaats of ouders? Of is het niet nodig hier een regel voor op te stellen en kan de gebruiker dat zelf wel verzinnen om ook te vragen?

Reply:
type moet altijd worden geleverd.